src/anchoredinterval.jl

```diff
@@ -284,3 +282,5 @@ canonicalize(target_type::Type{P}, p::P) where P <: Period = p
 function astimezone(i::AnchoredInterval{P, ZonedDateTime}, tz::TimeZone) where P
     return AnchoredInterval{P, ZonedDateTime}(astimezone(anchor(i), tz), inclusivity(i))
 end
+
+timezone(i::AnchoredInterval{P, ZonedDateTime}) where P = timezone(anchor(i))
```

Comment:
We no longer like extending functions with `import` and now prefer:
```suggestion
TimeZones.timezone(i::AnchoredInterval{P, ZonedDateTime}) where P = timezone(anchor(i))
```